sambacc/samba_cmds.py

```diff
@@ -99,7 +107,7 @@ def __repr__(self):
 
 smbd_foreground = smbd[
     "--foreground",
-    "--log-stdout",
+    smbd_stdout_opt(),
```

Comment:
This causes the environment variable to be checked at import time rather than when `smbd_foreground` is used. I'd rather switch smbd_foreground to a function so that the state of the module as imported is not dependent on the environment. This does mean changing the users of `smbd_foreground` but I think it's a small price to pay.

Reply:
I am updating the patch set with my python knowledge. Feel free to comment if I have to change something more.